build.gradle

```diff
@@ -877,17 +877,32 @@ task "bwcTestSuite"(type: RestIntegTestTask) {
     dependsOn tasks.named("fullRestartClusterTask")
 }
 
-task integTestRemote(type: RestIntegTestTask) {
+task integTestRemote (type: RestIntegTestTask) {
```

Comment:
Let's keep two separate gardle traget for `integTestRemote` and `singleClusterSanity` to avoid breaking builds till we release the support for passing multiple clusters during distro build.

We can switch between both targets in `integtest.sh` depending on whether a one cluster was passed as argument or two.

Reply:
I have made changed to both integtest.sh and the gradle task to work with both "single" and "multiple" cluster.
Once the changes are pushed to opensearch-build, we can remove this logic and make it only work with "multiple" clusters.

Reply:
Done.

Reply:
Thanks. 2 separate tasks do look cleaner.